### DISCUSSION TITLE
Avenir(?) du passe mission

### DISCUSSION THREAD

**Discussion by KirIcare** [Crownicles/Crownicles#1816] [Idées acceptées / déjà implémentées]
Depuis l'ajout de points aux récompenses des missions secondaires, le passe mission à encore moins d'utilité qu'au paravent (je l'ai jamais utilisé en 1an). Concrètement, qui va payer 10gemmes pour littéralement rien (déjà que y'en a qui utilise pas leurs gemmes même sur les options attirantes). Comme ajouter des fonctionnalités plutôt que d'en supprimer me tient à cœur, je préfère faire une suggestion pour le remplacer plutôt que d'en faire une pour crier à l'inutilité 😂.
Pourquoi ne pas remplacer le passe mission par un change mission ? Explications:
En l'achetant, on nous propose 3 missions secondaires à la place de l'actuelle et on en choisi une (ou on peut le faire un total random).
Ça permettrait de diversifier les moyens de gagner un top week, remplacer une mission qu'on était sûr de pas faire ou remplacer une mission en priant qu'on en obtient une qui donne plus de pts.
Pas besoin de limiter l'usage par semaine puisque il est déjà limité par la condition "avoir une mission secondaire"
A voir si le prix restera de 10 gemmes si la proposition est acceptée.

**Comment by BastLast** [comment]
pourquoi ne pas faire un "acheter une mission" dispo une fois par jour ?

**Reply by KirIcare** [reply]
On peux
Le but de ma suggestion était vraiment de faire évoluer le passe mission mais on peut partir sur tout autre chose

**Comment by BastLast** [comment]
je passe cette suggestion dans les suggestion approuvée puisque #1819  existe
